src/test/java/com/mongodb/hibernate/jdbc/MongoConnectionTests.java

```diff
@@ -111,24 +107,6 @@ void testNoOpWhenAutoCommitNotChanged(boolean autoCommit) throws Exception {
             @Nested
             class AutoCommitStateChangedTests {
 
-                @ParameterizedTest(name = "AutoCommit (true -> false): start transaction (successful: {0})")
-                @ValueSource(booleans = {true, false})
-                void testTryingToStartTransactionWhenAutoCommitChangedToFalse(boolean successful) throws SQLException {
```

Comment:
Let's add a test instead of `testTryingToStartTransactionWhenAutoCommitChangedToFalse`, which verifies that `setAutoCommit` does not start a transaction when the auto-commit state changes?

Reply:
Good point! Added. Thanks.